content/concepts/the-rippled-server/transaction-censorship-detection.md

```diff
@@ -23,20 +20,37 @@ At a high-level, here’s how the transaction censorship detector works:
 
     For as long as the transaction remains in the tracker, the detector continues to issue a warning message in the log every 15 ledgers, for up to five warning messages. After the fifth warning message, the detector issues a final [error message](#example-error-message) in the log and then stops issuing warning and error messages.
 
-    If you see these messages in your `rippled` server log, you may want to investigate, off-ledger, why other `rippled` servers are failing to include the transaction.
+    If you see these messages in your `rippled` server log, first verify that the messages are not [false positives](#potential-false-positives). If they are not false positives, you may want to investigate, off-ledger, why other `rippled` servers are failing to include the transaction.
```

Comment:
Nit: I would frame this slightly differently. Regardless of whether other servers are failing to include the transactions, you should investigate why other servers are failing to include the transaction, starting with the assumption that this is more likely to be a false positive (innocent bug) rather than malicious censorship.